contrib/backup/functions

```diff
@@ -177,7 +177,7 @@ function restore_zammad () {
       zammad run rake db:drop
       zammad config:set DISABLE_DATABASE_ENVIRONMENT_CHECK=0
     else
-      ${ZAMMAD_DIR}/bin/rake db:drop
+      DISABLE_DATABASE_ENVIRONMENT_CHECK=1 ${ZAMMAD_DIR}/bin/rake db:drop
```

Comment:
Without the variable, `rake` would refuse to drop the production DB on my test setup.